posts/2019-09-30-easily-deploy-graphql-apis-with-serverless-components.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Easily Deploy GraphQL APIs with Serverless Components
+description: "Introducing the AWS AppSync componnet that allows you to easily and quickly deploy GraphQL APIs on AWS, and integrate them with AWS Lambda, DynamoDB & others."
+date: 2019-09-17
+thumbnail: "https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/20191001-appsync-component/app-sync-component-thumbnail.png"
+heroImage: "https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/20191001-appsync-component/app-sync-component-header.png"
+authors:
+  - EslamHefnawy
+category:
+  - news
+---
+
+Ever since we released [Serverless Components](https://github.com/serverless/components), we've seen huge demand for an AppSync component. Especially that it is not natively supported in the Serverless Framework. Today, we're super thrilled to announce that [Serverless Components](https://github.com/serverless/components) now supports deploying GraphQL apps natively on AWS using the [aws-app-sync serverless component](https://github.com/serverless-components/aws-app-sync).
+
+We've built this component from scratch without any dependency on CloudFormation, making it a lot faster to deploy when compared to CloudFormation and Amplify. It also ships with sane defaults that makes it a lot simpler and easier to use without having to scroll through multiple pages of AWS documentation and copy/paste code. We've done the research for you, and we've packaged it all in this easy-to-use component. It comes with the following set of features:
```

Comment:
> We've built this component from scratch without any dependency on CloudFormation, making it a lot faster to deploy when compared to CloudFormation and Amplify. 

The [Serverless AppSync Component](https://github.com/serverless-components/https://github.com/serverless-components/aws-app-sync) does not rely on CloudFormation, making it a lot faster to deploy compared to CloudFormation and Amplify.